storage-node/src/commands/server.ts

```diff
@@ -219,16 +228,25 @@ Supported values: warn, error, debug, info. Default:debug`,
     const enableUploadingAuth = false
     const operatorRoleKey = undefined
 
-    await recreateTempDirectory(flags.uploads, TempDirName)
-
-    if (fs.existsSync(flags.uploads)) {
-      await loadDataObjectIdCache(flags.uploads, TempDirName, PendingDirName)
+    if (!flags.tempFolder) {
+      logger.warn(
+        'It is recommended to specify a unique file path for temporary files.' +
+          'For now a temp folder under the uploads folder will be used.' +
+          'In future this will be warning will become and error!'
```

Comment:
```
In future this will be warning will become and error!
```

Can you please rephrase that?

Reply:
Will rephrase the whole warning to:
```
You did not specify a path to the temporary directory. A temp folder under the uploads folder will
be used. In future passing an absolute path to a temporary directory with the tempFolder argument
will become mandatory.
```

WDYT?

Reply:
done in [a287fdc](https://github.com/Joystream/joystream/pull/5000/commits/a287fdc229c42642bb89b838ae0fe795f8c21ffe)